rand_distr/src/poisson.rs

```diff
@@ -44,43 +44,45 @@ pub enum Error {
     ShapeTooSmall,
 }
 
-impl Poisson {
+impl<N: Float> Poisson<N> {
```

Comment:
For other distributions, I require the same bounds on the constructor as on the `Distribution` implementation (in order to trap errors earlier).

Reply:
Done.